src/bundlers/node/nodeJsBinaryDependenciesBundler.ts

```diff
@@ -66,11 +71,13 @@ export class NodeJsBinaryDependenciesBundler implements BundlerInterface {
 
         for (const dependency of binaryDependencies) {
             try {
-                const { stdout, stderr } = await exec(
-                    "npx node-pre-gyp --update-binary --fallback-to-build --target_arch=arm64 --target_platform=linux --target_libc=glibc clean install " +
-                        dependency.name,
-                    { cwd: dependency.path },
-                );
+                const baseCommand =
+                    architecture == "arm"
+                        ? "npx node-pre-gyp --update-binary --fallback-to-build --target_arch=arm64 --target_platform=linux --target_libc=glibc clean install "
+                        : "npx node-pre-gyp --update-binary --fallback-to-build --target_arch=amd64 --target_platform=linux --target_libc=glibc clean install ";
```

Comment:
Maybe a switch?

Reply:
Managed to get around this by using the `execa` syntax.